src/Picqer/Financials/Exact/Connection.php

```diff
@@ -464,25 +464,25 @@ private function acquireAccessToken()
         }
     }
 
-    private function getDateTimeFromExpires($expires)
+    private function getTimestampFromExpiresIn($expiresIn)
```

Comment:
Might want to add a docheader, since you are here :)

Reply:
Well, I almost wanted to remove it completely because it's such a simple expression that might as well be inlined. It's used only once as well...

Reply:
For the sake of having everything documented, could you please add them?

It's only used once, but inline it wouldn't incorporate the checks that are done now.